index.d.ts

```diff
@@ -45,24 +45,28 @@ interface ESMSInitOptions {
 
   /**
    * #### Error hook
-   * 
+   *
    * Register a callback for any ES Module Shims module errors.
-   * 
+   *
    */
   onerror: (e: any) => any;
 
   /**
    * #### Resolve Hook
-   * 
+   *
    * Only supported in Shim Mode.
-   * 
+   *
    * Provide a custom resolver function.
    */
-  resolve: (id: string, parentUrl: string, resolve: (id: string, parentUrl: string) => string) => string | Promise<string>;
+  resolve: (
+    id: string,
+    parentUrl: string,
+    resolve: (id: string, parentUrl: string) => string
+  ) => string | Promise<string>;
```

Comment:
Prettier did this but I figured I'd leave it in since it's less likely to cause more diffs in the future (assuming most devs have Prettier running).